mava/components/building/datasets.py

```diff
@@ -125,6 +127,8 @@ class TrajectoryDatasetConfig:
     max_samples_per_stream: int = -1
     rate_limiter_timeout_ms: int = -1
     get_signature_timeout_secs: Optional[int] = None
+    prefetch_size: Optional[int] = 1
```

Comment:
Also maybe default it to what you've found to be a good value on GPUs

Reply:
I don't know if you should default to prefetching any number greater than 1 - its not always desired for a few reasons:
1. You might be using a large network so there is little VRAM available for an entire batch or more of data to be preloaded especially if the environment has lots of agents.
2. If the user is using a lot of actors and the CPU doesn't have enough resources to dedicate to prefetching, it can provide little benefit.